guide/glossary.md

```diff
@@ -111,13 +111,24 @@ Differentiate between
 
 Bitcoin with a capital *B* is typically associated with Bitcoin the protocol and payment network. It is also often used to refer to as the ecosystem as a whole when writing about it in general terms. Bitcoin with a lowercase “b” written as “bitcoin” is usually associated specifically with bitcoin as the currency.
 
-### Bitcoin Client
+### Bitcoin client
 
--What is considered a Bitcoin client? I'd suggest it's a software package comprising node / base layer wallet features. [Several other](https://coin.dance/nodes) clients exist besides Bitcoin Core. -Bosch
+Software that runs and connects to the Bitcoin network.
```

Comment:
A wallet connects to a remote node. Is the wallet then considered a client?

Reply:
Yes. SPV wallets are often referred to as light nodes, they are still technically a node as they are interacting with the network. Will be updating the node glossary definitions soon to reflect this.

Reply:
Unless is the key part of this definition is that the client software can not only connect but "run the network". So we'd have to make it clear where a client starts and ends... LDK modularises lightning nodes for example, but there is a core. Perhaps we can look at if there is something similar with the BDK.

_Technically speaking though a wallet can be considered a client because it connects to a remote server/node backend. So this distinction becomes a little tricky._

Reply:
Any software that runs and/or connects to the Bitcoin network is a client and can also be called a node. I changed this to and/or as it doesn't need to do both - my mistake.

 More specific distinctions can be made when you move down the tree into nodes where you have full / light nodes.